Updating docs examples (#2)
bytewax/helm-charts: README.md

# Bytewax Helm Chart

> * Installs the Python Stream Processing Library [Bytewax](https://bytewax.io/)

Reviewer: ```suggestion
* Installs the Python Stream Processing Library [Bytewax](https://bytewax.io/) and runs a Bytewax dataflow in on a Kubernetes cluster.
```

## Get Repo Info

```console
helm repo add bytewax https://bytewax.github.io/helm-charts
helm repo update
```

_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._

## Installing the Chart

To install the chart with the release name `my-release`:

```console
helm install my-release bytewax/bytewax
```

This version requires Helm >= 3.1.0.

## Uninstalling the Chart

> To uninstall/delete the my-release deployment:

Reviewer: ```suggestion
To uninstall/delete the `my-release` deployment:
```

### Example running basic.py obtained from a Configmap created by Helm

```yaml
configuration:
  pythonFileName: "basic.py"
  configMap:
    files:
      pattern: "examples/*"
      tarName:
```

### Example running pagerank.py obtained from a Configmap created by Helm

```yaml
configuration:
  pythonFileName: "examples/pagerank.py"
  configMap:
    files:
      pattern: 
      tarName: "examples.tar"
```

In this example, we store a tar file in the configmap. This is useful when your python script needs a tree of nested files and directories. 

Following our example, the tar file has this content:
```console
├── examples
│   ├── anomaly_detector.py
│   ├── basic.py
│   ├── events_to_parquet.py
│   ├── pagerank.py
│   ├── sample_data
│   │   ├── easy_count.txt
│   │   ├── graph.txt
│   │   ├── lyrics.txt
│   │   └── wordcount.txt
│   ├── search_session.py
│   ├── sleepyworkers.py
│   ├── subflow.py
│   ├── translator.py
│   ├── twitter_stream.py
│   ├── utils
│   │   ├── fake_events.py
│   │   ├── __init__.py
│   │   └── twitter.py
│   ├── wikistream.py
│   └── wordcount.py
```

> So, because that tar file is going to be extracted to the container working directory then the container is going to have that tree available to work with.

Reviewer: ```suggestion
Since that tar file is going to be extracted to the container working directory then the container is going to have that directory tree available to work with.
```

### Install Bytewax chart using a local copy and put your file(s) inside it

In this case, you will need to fetch the Bytewax chart to your machine and copy your python file(s) inside the chart directory.
Then, you can use the chart settings to generate a Configmap storing your file(s) and run it in the container.

There are the steps to include `my-code.py` and execute it:

1. Fetch Bytewax chart and decompress it
```console
$ helm repo add bytewax https://bytewax.github.io/helm-charts
$ helm repo update
$ helm fetch bytewax/bytewax
$ tar -xvf bytewax-0.1.0.tgz
```
2. Copy your file to the chart directory
```console
$ cp ./my-code.py ./bytewax/
```
3. Install Bytewax chart using your local copy
```console
$ helm upgrade --install my-dataflow ./bytewax \
  --set configuration.pythonFileName="my-code.py" \
  --set configuration.configMap.files.pattern="my-code.py"
```

### Create a Configmap before install Bytewax chart

In this option, you will need to provide a Configmap with your file(s) and then configure your chart values to use it.

> There are the steps to create a Configmap to store `my-code.py` and use it with the Bytewax chart:

Reviewer: ```suggestion
These are the steps to create a Configmap with `my-code.py` and use it with the Bytewax chart:
```
